### PR TITLE
To Do list: add & remove

### PR DESCRIPTION
**Pull request by Kayonga99**
## The following were implemented:

- No linter errors.
- Use correct git-flow.
- Document README.md file.
- Implement a function for adding a new task.
- Implement a function for deleting a task.
- Implement a function for editing task descriptions.

### CODE REVIEW

File: src/modules/classTaskList.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
+/*eslint-disable*/
```
Comment (ahmetbozaci) [inline src/modules/classTaskList.js:1]: - [x] Kindly don't disable linters. If it is giving error please fix it.

File: src/modules/refresh.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+/*eslint-disable*/
```
Comment (ahmetbozaci) [inline src/modules/refresh.js:1]: - [x] Kindly don't disable linters. If it is giving error please fix it.

File: src/modules/classTaskList.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+  updateStatus(taskID, status) {
+    this.data = this.data.map((obj) => {
+      if (obj.id === taskID) {
+        return { ...obj, completed: status };
+      }
+
+      return obj;
+    });
+    localStorage.setItem(storageName, JSON.stringify(this.data));
+  }
```
Comment (ahmetbozaci) [inline src/modules/classTaskList.js:56]: - [x] When click on the checkbox completed should be true. Kindly fix it.

Screenshot from 2022-07-09 00-23-54

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️


 @Kayonga99  ✋🏼 

#### Your project is excellent. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Nice to have 💯
- ✔️ Your git-flow is correct. ✔️
- ✔️ You set up linter files and no errors. ✔️
- ✔️ Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
- ✔️ I can add and edit to-do items.✔️
- ✔️ I can delete to-do items.✔️
- ✔️ If ı delete to-do item index updated.✔️
___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Review by FrederickMih — COMMENTED** [review]
Hi @Kayonga99 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**